api/src/test/java/org/pmiops/workbench/utils/TestMockFactory.java

```diff
@@ -73,7 +73,7 @@ public class TestMockFactory {
    * them bypassable and non-expirable. So we should default to this, unless required for product
    * reasons.
    *
-   * <p>RAS_LOGIN_GOV and CT_COMPLIANCE_TRAINING are bypassable and non-expirable for this reason.
```

Comment:
did you find that CT training is expirable in the DB?

Reply:
Yeah, local and prod both have expirable CT Compliance Training